src/view/uniprotkb/components/SequenceView.tsx

```diff
@@ -18,7 +18,7 @@ type Isoform = {
 };
 
 export type AlternativeProducts = {
-  commentType: Comment.ALTERNATIVE_PRODUCTS;
+  commentType: Comment;
```

Comment:
All comment types are unioned with `&` in. The way union works is that fields which are defined the same will be handled as being the same. So if you have `commentType: Comment` in 2 of them this will be treated as the same attribute.